src/Processors/Transforms/Streaming/AggregatingHelper.cpp

```diff
@@ -30,12 +30,19 @@ Chunk mergeBlocksToChunk(BlocksList && blocks)
     return merged_chunk;
 }
 
-Chunk convertToChunkImpl(AggregatedDataVariants & data, const AggregatingTransformParams & params, ConvertAction action)
+Chunk convertToChunkImpl(AggregatedDataVariants & data, const AggregatingTransformParams & params, AggregateStateType type)
```

Comment:
As mentioned `AggregateStateType` can be normalized with `std::optional<TrackingUpdatesType>`

Reply:
Seems make sense, can we rename `OnlyUpdates` -> `Updates`; `OnlyRetract` -> `Retract` ?